soc/arm/nxp_kinetis/k6x/power.c

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copytight (c) 2020 Nicolai Glud <nigd@prevas.dk>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr.h>
+#include <power/power.h>
+#include <drivers/pinmux.h>
+#include <fsl_port.h>
+#include <fsl_uart.h>
+#include <fsl_smc.h>
+
+
+#define LOG_LEVEL LOG_LEVEL_INF
+#include <logging/log.h>
+LOG_MODULE_REGISTER(soc);
+
+#ifdef CONFIG_SYS_POWER_SLEEP_STATES
+static void enable_uart(void)
+{
+	struct device *portb =
+		device_get_binding(CONFIG_PINMUX_MCUX_PORTB_NAME);
+
+	pinmux_pin_set(portb, 16, PORT_PCR_MUX(kPORT_MuxAlt3));
+	pinmux_pin_set(portb, 17, PORT_PCR_MUX(kPORT_MuxAlt3));
+
```

Comment:
The pinmux setting this is board specific and should be moved to the boards folder.